.goreleaser.yml

```diff
@@ -6,16 +6,29 @@ builds:
     flags:
       - -v
     ldflags: -X github.com/ava-labs/subnet-evm/plugin/evm.Version=v{{.Version}}
-    ignore:
-      - goos: darwin
-        goarch: 386
+    goos:
+      - linux
+      - darwin
+    goarch:
+      - amd64
+      - arm64
+    env:
+      - CGO_ENABLED=1
+      - CGO_FLAGS="-O -D__BLST_PORTABLE__"
```

Comment:
should we add this flag to any instructions we have to build from source @yulin-dong?

Reply:
this seems to be a good workaround for the pretty strange blst-related runtime error `Caught SIGILL in blst_cgo_init, consult <blst>/bindings/go/README.md.`  eg https://github.com/ava-labs/subnet-evm/actions/runs/3130327361/jobs/5080451590

Reply:
Currently our developer doc are more focused on avalanche-cli instead of subnet-evm directly. So I think we are fine not updating doc.